pkg/controller/chaosengine/chaosengine_controller.go

```diff
@@ -364,22 +313,15 @@ func (r *ReconcileChaosEngine) getChaosEngineInstance(engine *chaosTypes.EngineI
 
 // Get application details
 func getApplicationDetail(engine *chaosTypes.EngineInfo) error {
-	applicationInfo := &chaosTypes.ApplicationInfo{}
-	appInfo, err := initializeApplicationInfo(engine.Instance, applicationInfo)
```

Comment:
Dont we need this func `initializeApplicationInfo`. I think we are constructing the `appInfo` with some logic, can be helpful to keep it. WDYT?

Reply:
We were using appinfo struct at some places and cr.appinfo at other places. I made them consistent  bt using cr.appinfo at everyplace